### ISSUE TITLE
Errors when using the controls

### ISSUE THREAD

**Issue by benfysh** [mmende/homebridge-samsungtv-control2#61]
**Describe The Bug:**

When controlling the TV in the home app the TV no longer responds (it was previously working fine) and the errors shown in the logs below occur.

I have tried removing the TV and the cached accessory and adding it back to the home app and the issue persists.

**To Reproduce:**

1. Open the home app
2. Control the TV (Power Off)
3. Tile 'switches off' in home app
4. TV doesn't respond to control and remains on
5. TV 'switches on' in the home off

This also happens when you change the input in the home app or change the volume with the remote function in control centre.

**Expected behavior:**

The expectation is that the TV would power off and the tile in the home app remains on.
**Logs:**

```
(node:30472) UnhandledPromiseRejectionWarning: Error: Unable to handleEvent
    at /usr/local/lib/node_modules/homebridge-samsungtv-control/node_modules/samsung-remote-pin-paired/lib/Connection/SamsungTvConnection.js:130:13
    at /usr/local/lib/node_modules/homebridge-samsungtv-control/node_modules/samsung-remote-pin-paired/lib/Connection/Messages.js:30:31
    at new Promise (<anonymous>)
    at Messages.handle (/usr/local/lib/node_modules/homebridge-samsungtv-control/node_modules/samsung-remote-pin-paired/lib/Connection/Messages.js:29:12)
    at onMessageEmitter (/usr/local/lib/node_modules/homebridge-samsungtv-control/node_modules/samsung-remote-pin-paired/lib/Connection/SamsungTvConnection.js:45:12)
    at WebSocket.<anonymous> (/usr/local/lib/node_modules/homebridge-samsungtv-control/node_modules/samsung-remote-pin-paired/lib/Connection/SamsungTvConnection.js:157:9)
    at WebSocket.emit (events.js:315:20)
    at Receiver._receiver.onmessage (/usr/local/lib/node_modules/homebridge-samsungtv-control/node_modules/ws/lib/websocket.js:137:47)
    at Receiver.dataMessage (/usr/local/lib/node_modules/homebridge-samsungtv-control/node_modules/ws/lib/receiver.js:409:14)
    at Receiver.getData (/usr/local/lib/node_modules/homebridge-samsungtv-control/node_modules/ws/lib/receiver.js:347:12)
    at Receiver.startLoop (/usr/local/lib/node_modules/homebridge-samsungtv-control/node_modules/ws/lib/receiver.js:143:16)
    at Receiver.add (/usr/local/lib/node_modules/homebridge-samsungtv-control/node_modules/ws/lib/receiver.js:117:10)
    at Socket.emit (events.js:315:20)
    at addChunk (internal/streams/readable.js:309:12)
    at readableAddChunk (internal/streams/readable.js:284:9)
    at Socket.Readable.push (internal/streams/readable.js:223:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:188:23)
(node:30472) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
```
This error is also shown
```
[29/04/2021, 09:55:34] [homebridge-samsungtv-control] This plugin slows down Homebridge. The write handler for the characteristic 'Active Identifier' on the accessory 'Living Room TV 8419' didn't respond at all!. Please check that you properly call the callback! See https://git.io/JtMGR for more info
```

**Plugin Config:**

```json
        {
            "platform": "SamsungTVControl",
            "devices": [
                {
                    "usn": "uuid:795ce…",
                    "token": "eyJzZXNza…",
                    "name": "Living Room TV",
                    "inputs": [
                        {
                            "name": "HDMI 1",
                            "keys": "hdmi"
                        },
                        {
                            "name": "HDMI 2",
                            "keys": "hdmi1"
                        },
                        {
                            "name": "HDMI 3",
                            "keys": "hdmi2"
                        },
                        {
                            "name": "HDMI 4",
                            "keys": "hdmi3"
                        }
                    ]
                }
            ]
        },
```

**Screenshots:**
N/A

**Environment:**

* **Plugin Version**: v2.0.0
* **Homebridge Version**: v1.3.4
* **Node.js Version**: v14.16.0
* **NPM Version**: v6.13.4
* **Operating System**: Raspbian

**Comment by blackbasket** [comment]
Hi,

I have the same issue. Installed the plugin, it worked one time and now I get the errors.

Regards,
Marcel

**Comment by blackbasket** [comment]
Hi,

I tried again and getting this errors now:

`homebridge-samsungtv-control] Error: 
    at Accessory.sendCharacteristicWarning (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1874:14)
    at Timeout._onTimeout (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1416:14)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
[5/15/2021, 3:28:49 PM] [homebridge-samsungtv-control] Error: 
    at Accessory.sendCharacteristicWarning (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1874:14)
    at Timeout._onTimeout (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1416:14)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
[5/15/2021, 3:28:53 PM] [SamsungTVControl] TV-Gerät - GET Active
[5/15/2021, 3:28:53 PM] [SamsungTVControl] TV-Gerät - GET Mute
[5/15/2021, 3:28:53 PM] [SamsungTVControl] TV-Gerät - GET Volume
[5/15/2021, 3:28:55 PM] [homebridge-samsungtv-control] This plugin slows down Homebridge. The read handler for the characteristic 'Mute' on the accessory 'TV-Gerät DC52' didn't respond at all!. Please check that you properly call the callback! See https://git.io/JtMGR for more info.
[5/15/2021, 3:28:55 PM] [homebridge-samsungtv-control] Error: 
    at Accessory.sendCharacteristicWarning (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1874:14)
    at Timeout._onTimeout (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1431:16)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
[5/15/2021, 3:28:55 PM] [homebridge-samsungtv-control] This plugin slows down Homebridge. The read handler for the characteristic 'Volume' on the accessory 'TV-Gerät DC52' didn't respond at all!. Please check that you properly call the callback! See https://git.io/JtMGR for more info.
[5/15/2021, 3:28:55 PM] [homebridge-samsungtv-control] Error: 
    at Accessory.sendCharacteristicWarning (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1874:14)
    at Timeout._onTimeout (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1431:16)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
[5/15/2021, 3:29:02 PM] [homebridge-samsungtv-control] This plugin slows down Homebridge. The read handler for the characteristic 'Mute' didn't respond at all!. Please check that you properly call the callback! See https://git.io/JtMGR for more info.
[5/15/2021, 3:29:02 PM] [homebridge-samsungtv-control] This plugin slows down Homebridge. The read handler for the characteristic 'Volume' didn't respond at all!. Please check that you properly call the callback! See https://git.io/JtMGR for more info.
[5/15/2021, 3:29:03 PM] [SamsungTVControl] TV-Gerät - GET Active
[5/15/2021, 3:29:03 PM] [SamsungTVControl] TV-Gerät - GET Mute
[5/15/2021, 3:29:03 PM] [SamsungTVControl] TV-Gerät - GET Volume
[5/15/2021, 3:29:03 PM] [SamsungTVControl] TV-Gerät - GET Active
[5/15/2021, 3:29:03 PM] [SamsungTVControl] TV-Gerät - GET Mute
[5/15/2021, 3:29:03 PM] [SamsungTVControl] TV-Gerät - GET Volume
[5/15/2021, 3:29:06 PM] [homebridge-samsungtv-control] Error: 
    at Accessory.sendCharacteristicWarning (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1874:14)
    at Timeout._onTimeout (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1416:14)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
[5/15/2021, 3:29:06 PM] [homebridge-samsungtv-control] Error: 
    at Accessory.sendCharacteristicWarning (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1874:14)
    at Timeout._onTimeout (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1416:14)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
`

Regards,
Marcel

**Comment by benfysh** [comment]
This was resolved for me in v2.0.2 I did have to remove and then add back my TV and I think I used a different pairing method than when I first installed.
